### ISSUE TITLE
Support custom report/output directory

### ISSUE THREAD

**Issue by calebcartwright** [xd009642/tarpaulin#245]
First I just want to say thank you for this awesome project! ❤️ 

I was wondering if you'd be receptive to a new option to allow the user to control where the Xml/Html/etc. report files are written?

The current behavior writes the files to the cwd, so if I run `cargo tarpaulin -o Xml -o Html` I end up with:
```bash
./cobertura.xml
./tarpaulin-report.html
```

I'd love to have a new option (perhaps something like `--output-dir` or `--report-dir`) that would let me specify a target directory to emit those reports. For example:
`cargo tarpaulin -o Xml -o Html --output-dir .reports`
would give me:
```bash
./.reports/cobertura.xml
./.reports/tarpaulin-report.html
```

I'd be happy to take a pass at implementing this and submit a PR as well

**Comment by xd009642** [comment]
Sure I see nothing wrong with that feature. It is also possible to run tarpaulin from a folder called reports and then use the root argument to see the directory it's run from to be a different one.

If you do a PR for this is will be pretty simple, config module changed to add the option, main.rs and then all the report types in the report module changed ☺️

**Comment by calebcartwright** [comment]
Thanks for both the alternative and the pointers on where to look in the code!

PR opened with a first pass at this, let me know if there's any improvements/changes/etc. needed 👍

**Comment by calebcartwright** [comment]
Closing as this is now available in 0.8.7!
